src/pathfinding_service/model/channel_view.py

```diff
@@ -18,6 +19,11 @@
 
 
 class Interpolate:  # pylint: disable=too-few-public-methods
```

Comment:
Just a general question: might it be helpful to use numpy here? I'm not sure if it's worth the dependency, but interpolations should be implemented there and probably more performant.

Reply:
It's certainly faster and more mature. I just wanted to avoid adding a problematic dependency. Numpy needs quite a lot of build tools if there's not prebuilt wheel for your system.

Reply:
Yeah, we can still add this later.